youtube_dl/extractor/ndtv.py

```diff
@@ -22,21 +26,22 @@ class NDTVIE(InfoExtractor):
             'upload_date': '20131208',
             'duration': 1327,
             'thumbnail': r're:https?://.*\.jpg',
-        },
+        }
```

Comment:
It's been requested to make tests `only_matching` not to remove them.

Reply:
Oh... sorry.  Adding them back.